src/pages/Product/Product.jsx

```diff
@@ -0,0 +1,34 @@
+import { ProductCard,Filter,Footer } from "../../Components";
+import "./product.css"
+import { useProducts } from "../../context/ProductContext";
+
+
+
+const Product=()=>{
+    const {data}=useProducts();
+    console.log(data);
+    return(
+        <>
+        {console.log(data)}
+        <div className="product-grid padding-lg">
+            <aside className="product-filter">
+                <Filter/>
+            </aside>
+            {/* Product List */}
+            <div className="main-product">
+                {/* Heading */}
```

Comment:
Isn't this comment redundant? Cause the coded can be easily identified as a heading without the comment too!

Reply:
Removed the comment